Backend/api/views/transaction_view.py

```diff
@@ -0,0 +1,68 @@
+from requests import Session
+from api.models.address import Addresses
+from api.models.transactions import Transactions
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.views import APIView
+from buidl.tx import Tx, TxOut, TxIn, RedeemScript
+from rest_framework.response import Response
+from rest_framework import status
+
+class CreateTransactionView(APIView):
+    
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+    
+
+    def post(self, request):
+    
+        address = Addresses.objects.get(address_generated=request.data['address'])
+        address_generated = address.address_generated
+    
+        transaction_request = Session().get(url=f"https://blockstream.info/testnet/api/address/{address_generated}/utxo")
+        amount_to_send = request.data['amount']
+        recipient = request.data['recipient']
+        utxo = 0
+        txn_inputs = []
+        txn_input_ids = []
+        for tran in transaction_request.json():
+            if amount_to_send + 1000 >= utxo:
+                txn_input_ids.append(tran['txid'])
+                utxo = utxo + tran['value']
+                pub_list = list(address.redeem_script.split(' '))
```

Comment:
This is not needed, since the redeem script is saved in the database, you can just do address = Addresses.objects.get(address_generated=request.data['address']), then address.redeem_script to get the redeemscript

Reply:
This has been resolved in the sign-transaction branch. I will raise a PR soon

Reply:
okay, good.